### DISCUSSION TITLE
Broadcom HCI vendor commands

### DISCUSSION THREAD

**Discussion by ndeadly** [seemoo-lab/nexmon#634] [Q&A]
I'm interested in sending some [undocumented Broadcom HCI commands](https://github.com/vigoss30611/buildroot-ltc/blob/31cf5593a5bb4de4608425886e93f4be628f87f4/system/app-specs-bt/include/app_tm_vsc.h#L27C24-L28) for Triggered Broadcom Fast Connect (TBFC) to the CYW43439 
 chip on a raspberry pi pico w. The datasheet for the CYW43439 states that it supports TBFC, so I assumed I should be able send these commands in order to utilise TBFC features. However, sending the commands with the pico sdk results in an UNKNOWN_HCI_COMMAND (0x01) status, so I assume they must be disabled by default.

Any idea if it would be possible to create patches to enable such commands, and how straightforward (or not) it might be?

**Comment by jlinktu** [comment]
`nexmon` is focused on the Wi-Fi part of the BCM43/CYW43 chip series.
You might wan't to check out [InternalBlue](https://github.com/seemoo-lab/internalblue) regarding the Bluetooth side of those chips.

**Reply by ndeadly** [reply]
Oh my bad, I assumed it was all part of the same thing.
